test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Routing/ODataRoutingTest.cs

```diff
@@ -420,7 +420,7 @@ public async Task RoutesCorrectly(string httpMethod, string uri, string expected
         [Theory]
         [InlineData("CollectionOfPrimitiveTypeFunction(intValues=@p)?@p=[1,2,4,7,8")] // missing "]"
         [InlineData("CanMoveToAddress(address=@p)?@p={\"@odata.")] // not valid complex payload
-        [InlineData("EntityTypeFunction(product=@p)?@p={\"@odata.type\":\"Microsoft.AspNet.OData.Test.Routing.Product\",\"id\":9,\"Name\":\"Phone\"}")] // should be "ID"
+        [InlineData("EntityTypeFunction(product=@p)?@p={\"@odata.type\":\"Microsoft.AspNet.OData.Test.Routing.Product\",\"id1\":9,\"Name\":\"Phone\"}")] // should be "ID"
         public async Task RoutesIncorrectly_ForBadFunctionParameters(string uri)
```

Comment:
why changing from "id" to "id1"? It doesn't look like a case-sensitive? it's a different name

Reply:
This is an existing test that was meant to fail when properties are case-insensitive. Since we support case-insensitive property binding by default, I opted to change the property name.

Reply:
@KenitoInc Wasn't the right action to remove the test case instead? Consider the comment at the end of the test case - `// should be "ID"`. Isn't that misleading now?

Reply:
I believe that the wrong property name counts as "Bad function parameter" (see test name)
`// should be "ID"` indicates that the correct property name should have been `ID`

Reply:
Strictly speaking, relatively to `ID`, `id1` is not a wrong property name - it's a different property name. It doesn't count as bad function parameter.